crates/storage-query-datafusion/src/journal/row.rs

```diff
@@ -103,3 +112,15 @@ fn deserialize_invocation_request(entry: &EnrichedRawEntry) -> Option<InvokeRequ
         _ => None,
     }
 }
+
+fn deserialize_sleep_entry(entry: &EnrichedRawEntry) -> Option<SleepEntry> {
```

Comment:
I'm not sure if that would be a good home for this. I'd prefer to leave this as is for now to avoid distracting myself by what might end up being a larger refactoring :)